chapters/interface.tex

```diff
@@ -383,30 +382,30 @@ \section{Interface Compatibility or Subtyping}\doublelabel{interface-compatibili
     is not possible to have} inner \emph{and/or} outer \emph{as part of
     a redeclare)}.
   \item
-    If B is final A must also be final and have the same semantic
+    If \lstinline!B! is final \lstinline!A! must also be final and have the same semantic
     contents.
   \item
-    The number of array dimensions in A and B must be matched.
+    The number of array dimensions in \lstinline!A! and \lstinline!B! must be matched.
   \item
     Conditional components are only compatible with conditional
     components. The conditions must have equivalent contents (similar as
-    array sizes -- except there is no ``:'' for conditional components).
+    array sizes -- except there is no \lstinline!:! for conditional components).
     \emph{For a redeclaration of an element the conditional part is
     inherited from the original.}
   \item
-    A function class is only compatible with a function class, a package
+    \lstinline!A! function class is only compatible with a function class, a package
```

Comment:
This should be without lstinline, i.e. "A function class"

Reply:
Fixed.